pyproject.toml

```diff
@@ -145,6 +145,7 @@ benchmarking = [
     "baybe[chem]",
     "baybe[onnx]",
     "baybe[simulation]",
+    "boto3>=1.0.0",
```

Comment:
There was a jump from 0.0.22 to 1.0.0 in 2015 which usually indicates a major breaking change and since the current version is 1.35.64 I thought the choice would be reasonable. We could also drop the version, but I would want to try the smallest version just to be sure.

Reply:
Ok, 2015, that's really long ago 😅 Then I think it's absolutely OK to have this as the lower limit. But you could in fact add *upper* limits to both `benchmark` dependencies, i.e. `<2` for `boto3` and `<4` for `GitPython`. This avoids sudden breakdowns of our package just like it happened with `numpy==2.0`, which probably ripped 90% of all scientific packages apart 😄

Reply:
That is probably a good idea, will add it. Thanks 😃